python.md

```diff
@@ -0,0 +1,87 @@
+# Use faster BLAS routines for SciPy and NumPy
+
+The default SciPy and NumPy binary installations with `pip3 install numpy scipy`
+are configured to use OpenBLAS. On Ubuntu `apt install python3-numpy
+python3-scipy` will install NumPy and SciPy with BLAS and LAPACK libraries.
+The default installations of SciPy and NumPy are easy to setup and well
+tested.
+
+Some workloads will benefit from using BLIS. Benchmarking SciPy and NumPy
+workloads with BLIS might allow to identify additional performance improvement.
+If NumPy and SciPy detect the presence of the BLIS library, they will use BLIS
+in priority over the same functionality from BLAS and OpenBLAS.
+OpenBLAS or LAPACK libraries need to be installed along BLIS to provide LAPACK
+functionality.
+
+To install the prerequisites to build SciPy and NumPy with BLIS and OpenBLAS on
+Ubuntu and Debian:
```

Comment:
Do you need to? Can't you just install and use update-alternatives?
``
update-alternatives --config libblas.so.3-aarch64-linux-gnu
```

Reply:
There are several ways of installing and modifying the default BLAS.
I have not experimented yet with changing BLAS with `update-alternatives`.
My understanding is that you would still need to `apt install libblas-dev`.

Reply:
Updated instructions to use alternatives.  Much easier to install that way.
Thanks for the suggestion!